internal/debug.go

```diff
@@ -14,33 +14,33 @@ var (
 
 // DebugLogger provides an interface to call the logging functions
 type DebugLogger interface {
```

Comment:
Behaves just like `fmt.Println()` or `log.Println()`
for example: `logger.LogInfo("random pre-fix string", 10) # random pre-fix string 10`